### PR TITLE
Pecan background importer

### PR DESCRIPTION
**Pull request by javierarce**
cc @xavijam

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4994/))

File: lib/assets/javascripts/cartodb/common/background_polling/models/pecan_model.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    var skew        = stats.skew;
+    var distinct    = stats.distinct;
+    var count       = stats.count;
+    var null_ratio  = stats.null_ratio;
+    var dist_type   = stats.dist_type;
+    var calc_weight = cdb.CartoCSS.getWeightFromShape(dist_type);
+
+    var distinctPercentage = (distinct / count) * 100;
+
+    console.log("%cAnalyzing %c" + name, "text-decoration:underline; font-weight:bold", "text-decoration:underline; font-weight:normal");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/background_polling/models/pecan_model.js]: could we change this for ```cdb.log.info()```

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_card.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+  _CARD_WIDTH: 288,
+  _CARD_HEIGHT: 170,
+  _TABS_PER_ROW: 3,
+  _GET_BBOX_FROM_THE_GEOM: true,
+
+  tagName: "li",
+  className: "GalleryList-item MapsList-item js-card",
+
+  events: cdb.core.View.extendEvents({
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_card.js]: I don't think cdb.core.View has any event from the beginning, so we don't need this.

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js [hunk 3]
```diff
@@ -73,426 +50,15 @@ module.exports = BaseDialog.extend({
...
-      ViewFactory.createByTemplate('common/templates/loading', {
-        title: 'Applying style…',
-        quote: randomQuote()
-      })
-    );
-
-    if (this._check()) {
-      this._panes.active('loading');
+    if (this._check() && this.backgroundPollingModel.canAddAnalysis()) {
+      this.backgroundPollingModel.removeAnalysis();
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_view.js]: You should not need to remove the previous analysis, shouldn't you?

File: lib/assets/javascripts/cartodb/common/background_polling/models/pecan_model.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/background_polling/models/pecan_model.js]: Sure

**Comment by xavijam** [comment]
Meanwhile we test the functionality, we should try to get back to life those commented tests.
And, in overall, looooooks awesome.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4996/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4997/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4998/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5007/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5011/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5013/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5018/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5019/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5020/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5021/))

**Pull request merged**